### DISCUSSION TITLE
Rotex HSPU working and some observations

### DISCUSSION THREAD

**Discussion by jayrockk** [raomin/ESPAltherma#283] [Show and tell]
Hi

First of all, thanks for this great piece of software!

I'm running a 2015 Rotex HPSU with HPSU_ULTRA. It works fine. One observation is that {0x60,7,105,2,1,"DHW setpoint"} consistently shows 1 deg higher than the RoCon display. I haven't tested all of the values, though, so I cannot say whether other values are off.

My HW is an M5StickC Plus, which happily powers from the 8 pin X10A. I removed the USB cable after installation.

Cheers,

Jo

![PXL_20230513_124007343](https://github.com/raomin/ESPAltherma/assets/614481/82ce8853-02e8-4985-89ff-6528358f0216)
![PXL_20230513_165351563](https://github.com/raomin/ESPAltherma/assets/614481/2bf5c4f7-993e-4488-98fe-393b82ceaae3)

**Comment by KalaNagHTD** [comment]
The one degree offset is normal, due to the two controller boards (Daikin A1P and Rotex PM1) inside the HPSU which handle the DHW independently. Each board has its own temp sensor for DHW. The Display is connected to the P1M, the X10A is on the A1P. Curious, isn‘t it ;-)

**Comment by bgcngm** [comment]
Are these pictures from the board in the external unit? What is the pump that you had to select in the ESPAltherma configuration?

**Reply by jayrockk** [reply]
The pictures are taken from the indoor unit.

Configuration is described in the first post.

**Reply by bgcngm** [reply]
Thanks for your quick reply. I am confused because I saw a guy on a YouTube video installing it on the external unit.

As for the configuration I was only asking about the #include line to select the heat pump.

**Reply by jayrockk** [reply]
I just browsed the code and it looks like HSPU_ULTRA was removed since I installed it. I don't know when it went removed, so I guess you would have to speak with the developers.

EDIT: First of all, apologies, the file is called ALTHERMA(HPSU6_ULTRA).h

I've checked and I'm on commit 81096cd, In this commit,  [ALTHERMA(HPSU6_ULTRA).h](https://github.com/raomin/ESPAltherma/blob/81096cd709cb32e4b78b452cfc6d667dd48c8b3d/include/def/ALTHERMA(HPSU6_ULTRA).h) still exists. I still can't figure out when it was removed, the file [history](https://github.com/raomin/ESPAltherma/commits/0aaf1749e12e3ba3083ef445c9cf936aa490b424/include/def/ALTHERMA(HPSU6_ULTRA).h) doesn't indicate anything.

**Comment by jayrockk** [comment]
Check the diff on setup.h [here](https://github.com/raomin/ESPAltherma/commit/80a7acdb2d4ec00d97973e964427cc9076bb0859#diff-2b699b4be659be53c5d13ba01b3d2a9fcfd86f5549996ffcab6b08d1673f4a21), it looks like it has been replaced with Altherma(EGSQH-A series 10kW GEO2).
